tests/integration/test_fugue.py

```diff
@@ -32,11 +33,20 @@ def test_simple_statement():
     return_df = result["result"].as_pandas()
     assert_eq(return_df, pd.DataFrame({"a": [1], "b": ["world"]}))
 
+    result = fugue_sql.fsql(
+        "SELECT * FROM df WHERE a > 0 YIELD DATAFRAME AS result",
+        df=pdf,
+    ).run(client)
+
+    return_df = result["result"].as_pandas()
+    assert_eq(return_df, pd.DataFrame({"a": [1], "b": ["world"]}))
+    client.close()
```

Comment:
Just noticed, we shouldn't need to close the client here manually as the fixture should do clean up on its own:

```suggestion
    client.close()
```